pages/agent/gcloud.md.erb

```diff
@@ -0,0 +1,202 @@
+                                                __/ |
+ http://buildkite.com/agent                    |___/
+2017-02-16 03:34:10 NOTICE Starting buildkite-agent v2.3.2 with PID: 5
+2017-02-16 03:34:10 NOTICE The agent source code can be found here: https://github.com/buildkite/agent
+2017-02-16 03:34:10 NOTICE For questions and support, email us at: hello@buildkite.com
+2017-02-16 03:34:10 INFO   Registering agent with Buildkite...
+2017-02-16 03:34:11 INFO   Successfully registered agent "buildkite-agent-3220167863-1cvr5" with meta-data []
+2017-02-16 03:34:11 INFO   Connecting to Buildkite...
+2017-02-16 03:34:11 INFO   Agent successfully connected
+2017-02-16 03:34:11 INFO   You can press Ctrl-C to stop the agent
+2017-02-16 03:34:11 INFO   Waiting for work...
+```
+
+You’re successfully running a Buildkite Agent!
+
+To run more than one agent you can increase `replicas`:
+
+```shell
+$ kubectl scale --replicas=5 deployment buildkite-agent
+deployment "buildkite-agent" scaled
+
+$ kubectl get pods
+NAME                               READY     STATUS    RESTARTS   AGE
+buildkite-agent-3220167863-0m269   1/1       Running   0          24s
+buildkite-agent-3220167863-0mtps   1/1       Running   0          24s
+buildkite-agent-3220167863-1cvr5   1/1       Running   0          3m
+buildkite-agent-3220167863-9psz9   1/1       Running   0          24s
+buildkite-agent-3220167863-m3nm2   1/1       Running   0          24s
+```
+
+To run builds from private repositories you may want to store [an SSH key for the agent](/docs/agent/ssh-keys) in a secret and map it into the containers:
+
+```shell
+# Use an existing keypair, or generate a new one with something like:
+$ ssh-keygen -t rsa -b 2048 -N "" -C builkdite-agent -f id_rsa
+Generating public/private rsa key pair.
+Your identification has been saved in id_rsa.
+Your public key has been saved in id_rsa.pub.
+...
+
+# Create a secret containing the ssh keys:
+$ kubectl create secret generic buildkite-agent-ssh \
+    --from-file id_rsa=id_rsa \
+    --from-file id_rsa.pub=id_rsa.pub
+secret "buildkite-agent-ssh" created
+
+# Change the spec to include new volumes to map the ssh key into place (because
+# this is Docker the git and ssh processes are running as root):
+$ kubectl edit deployment buildkite-agent
+...
+spec:
+  containers:
+  - ...
+    volumeMounts:
+    - name: ssh-key
+      subPath: id_rsa
+      mountPath: /root/id_rsa
+      readOnly: true
+    - name: ssh-key
+      subPath: id_rsa.pub
+      mountPath: /root/id_rsa.pub
+      readOnly: true
+    ...
+  volumes:
+  - name: ssh-key
+    secret:
+      secretName: buildkite-agent-ssh
+      defaultMode: 256
+  ...
+```
+
+To [configure](/docs/agent/configuration) the agent further you might like to create a [config map](https://kubernetes.io/docs/user-guide/configmap/) and volume mount it over the default agent configuration file in `/buildkite/buildkite-agent.cfg`.
```

Comment:
'Can' instead of 'might like to' - we're here to help them get shit done! We want them to feel empowered to do things, not 'oh yeah maybe I will sometime'.